frontend/src/components/CurrencyInputField.tsx

```diff
@@ -11,18 +11,27 @@ import React from "react";
 interface CurrencyInputFieldProps {
     onChange: any;
     value: StringOrNumber | undefined;
+    min?: number;
+    max?: number;
 }
 
 export default function CurrencyInputField(
     {
         onChange,
         value,
+        min,
+        max,
     }: CurrencyInputFieldProps,
 ) {
+    let minAmount = min ? min! : 0;
+    let maxAmount = max ? max! : Number.MAX_SAFE_INTEGER;
```

Comment:
I _think_ there is a dedicated operator in TS for what you are trying to achieve here.

```suggestion
    let minAmount = min || 0;
    let maxAmount = max || Number.MAX_SAFE_INTEGER;
```

Reply:
nice,